factory-reset-lvm

```diff
@@ -147,6 +147,8 @@ lvm vgchange -a n
 lvm vgremove -y sailfish
 lvm pvremove -y $PHYSDEV
 
+dd if=/dev/zero of=$PHYSDEV bs=1M
```

Comment:
Should't it be made conditionally? I mean like checking /.wipe file or so. So it could be triggered from OS if it's needed.